frontend/src/app/resource-form/resource-form.component.ts

```diff
@@ -103,4 +105,15 @@ export class ResourceFormComponent implements OnInit, OnDestroy {
         }
       });
   }
+  
+  // Automatically set values of CPU and Memory if GPU is 1
+  checkGPU(gpu: string) {
+    if (gpu == "none") {
+      this.readonlySpecs = false;
+    } else {
+      this.readonlySpecs = true;
+      this.formCtrl.get("cpu").setValue("5");
+      this.formCtrl.get("memory").setValue("96Gi");
```

Comment:
Same question here re. imperative form here vs. declarative via template. Though it makes more sense to me in this case.